### ISSUE TITLE
How to list currently logged in & previously logged in users?

### ISSUE THREAD

**Issue by JB-Prof** [ltsp/community#148]
Hello,

Previously with LTSP I used to use the 'w' or 'who' command to determine who is currently logged into the system, and the 'last' command to determine who was previously logged in, for how long, where they logged in from, etc.

I tried it on my Ubuntu-Mate 18.04 LTSP server and neither command is displaying the client login information -- only those who directly logged into the server (which makes sense).

How can I obtain this information please?

Thank you.

**Comment by alkisg** [comment]
For security reasons, LTSP doesn't use SSH logins anymore, but SFTP connections, see https://github.com/ltsp/ltsp/issues/128.
This makes it easier to restrict SFTP connections to a chroot, to completely disallow server access to the clients etc.

So, `w` and `who` don't consider SFTP connections a "login", even though SSH properly uses PAM for SFTP connections.
To list the remote users, more intelligent tools like `loginctl` should be used.

**Comment by JB-Prof** [comment]
Hello,

Thank you for the reply.  I am looking for this information because sometimes we need to reboot the server or see who forgot to log out and these commands helped with that.  Also we do need to provide some reporting at times of who logged in when, etc.

I tried to turn on SFTP logging to determine who is connected and when they log in / out but the log levels for sftp are either massive (every file transferred gets logged) or nothing about the user.  So that didn't work.

I tried to use loginctl as recommended but it was confusing as:
`loginctl session-status` and `loginctl list-users` gave me different answers of who is currently connected to the server.  I am probably not understanding the command very well.  It also gives me no information on who logged in before but has now logged out.

One option that I came up with that I am sure is totally not elegant and needs improvement is this:
`journalctl --since yesterday -r | grep 'sshd' | grep 'session closed\|session open' | grep -v -i 'cron\|lightdm' | grep -v myusername`

This will show me when someone's session was opened or closed but I've excluded cron and lightdm as they are both logged, and I've excluded my own username so that it makes the results a bit less crowded as I log in and out a lot.  I have no idea what I'm doing on the command line so I imagine there is a better way to write that!  Ideally it would be in some sort of script that can make sense of the results since for now I need to eye-ball the results to make sense of it.

I put this here so in case other LTSP users are also looking for such information.

Thank you.
